target/linux/ramips/base-files/lib/upgrade/platform.sh

```diff
@@ -24,6 +24,8 @@ platform_check_image() {
 	all5002|\
 	all5003|\
 	mediatek,ap-mt7621a-v60|\
+	netbot|\
```

Comment:
Really? I told you three times now to use the compatible from the dts as userspace boardnames. You have done it right in 02_network but kindly ignored this one.

Reply:
Forgive my bad english understanding. you mean we should use the full name from dts instead of the short name?

Reply:
exactly.